corehq/ex-submodules/pillowtop/migrations/0007_copy_xforms_checkpoint.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from corehq.util.django_migrations import skip_on_fresh_install
+from pillowtop.models import KafkaCheckpoint
+
+
+@skip_on_fresh_install
+def copy_xforms_checkpoint(*args, **kwargs):
+    old_checkpoint_id = 'xform-pillow-xforms_2016-07-07-report_xforms_20160824_1708-hqusers_2020-03-25'
+    new_checkpoint_id = 'xform-pillow-xforms_2016-07-07-hqusers_2020-03-25'
+    for old in KafkaCheckpoint.objects.filter(checkpoint_id=old_checkpoint_id):
+        KafkaCheckpoint.objects.create(
+            checkpoint_id=new_checkpoint_id,
+            topic=old.topic,
+            partition=old.partition,
+            offset=old.offset,
+            doc_modification_time=old.doc_modification_time,
```

Comment:
Do you know where this column is used? I could see the current time being better here, depending on where it is referenced.

Reply:
I checked and didn't see anywhere.  Context is I think [lost to fogbugz](https://github.com/dimagi/commcare-hq/pull/21629) if it's not in someone's head.  There's also a `last_modified` that's `auto_now=True` which will capture the current time.  This field appears to store the modification time of the doc (xform) which triggered the last checkpoint update.  I'm not sure what that provides that `last_modified` doesn't, but since they're subtly different, I thought this made the most sense.

Reply:
that makes sense to me